packages/app/components/Today/ChatbotToday.tsx

```diff
@@ -172,7 +171,7 @@ export const ChatbotToday: React.FC = () => {
       <ChatbotContainer>
         <Card
           title="Course chatbot"
-          className=" flex h-[85vh] w-full flex-col overflow-y-auto sm:h-[90vh]"
+          className=" flex h-full max-h-[750px] w-full flex-col overflow-y-auto"
```

Comment:
yo um did you mean to do this? i think this is going to re-introduce a bug where the chatbot window height is going to be based on the number of queues